src/services/followMainFeedFollowing.js

```diff
@@ -1,23 +1,25 @@
-const stream = require("getstream");
+const stream = require('getstream');
+const UserService = require('../services/postgres/UserService');
 
 const followMainFeedFollowing = async (userId, userIds) => {
   if (!userIds || userIds.length == 0) {
     return;
   }
 
   const cs = stream.connect(process.env.API_KEY, process.env.SECRET);
+  const userSevice = new UserService();
   const userAdmin = await userSevice.getUserAdmin(process.env.USERNAME_ADMIN);
 
   userIds.push(userAdmin.user_id);
   const payload = userIds.map((ui) => {
     return {
       source: `main_feed_following:${userId}`,
-      target: `user_excl:${ui}`,
+      target: `user_excl:${ui}`
     };
   });
   return await cs.followMany(payload);
 };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It's not necessary to use `return await` since the function is already async. You can just return the promise directly.
```diff
- return await cs.followMany(payload);
+ return cs.followMany(payload);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->